tests/drivers/build_all/video/app.overlay

```diff
@@ -148,8 +148,6 @@
 			test_i2c_ov7675: ov7675@21 {
 				compatible = "ovti,ov7675";
 				reg = <0x21>;
-				reset-gpios = <&test_gpio 0 0>;
-				pwdn-gpios = <&test_gpio 1 0>;
```

Comment:
Doesn't it work anymore? If it still works you can keep these lines to test extra code.